config/assisted-service/assisted-service-configmap.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
```

Comment:
can't we use https://github.com/openshift/assisted-service/blob/master/openshift/template.yaml?
It will be the third file defining those parameters

Reply:
I wasn't sure if we needed to archive this file or even the files under ./bundle. This file has been removed.